CommonLib/Error/GREYError+Private.h

```diff
@@ -60,9 +60,9 @@ GREY_EXTERN NSString *const kErrorFunctionNameKey;
 GREY_EXTERN NSString *const kErrorUserInfoKey;
 
 /**
- * Key used to retrieve the bundle ID from an error object dictionary.
+ * Key used to retrieve the error info dictionary from an error object dictionary.
  */
-GREY_EXTERN NSString *const kErrorBundleIDKey;
+GREY_EXTERN NSString *const kErrorErrorInfoKey;
```

Comment:
Where is this used? errorInfo can be obtained without this change. Please revert if not needed.

Reply:
Only inside GREYError.m. Removed it.